accel-pppd/ifcfg.c

```diff
@@ -56,6 +57,13 @@ void ap_session_ifup(struct ap_session *ses)
 		ses->ifname_rename = NULL;
 	}
 
+	if (ses->vrf_name) {
+		if (ap_session_set_vrf(ses)) {
```

Comment:
Let's make it consistent with interface rename and netns, ex: ap_session_set_vrf(ses, ses->vrf_name, -1)